### PR TITLE
Solve the problem of garbled characters

### PR DESCRIPTION
**Pull request by lhDream**
Solve the problem that CJK characters cannot be packaged and installed packages and garbled characters. #202

### CODE REVIEW

**Comment by lhDream** [comment]
https://stackoverflow.com/questions/51066207/utf-8-characters-not-displaying-correctly-in-inno-setup
`The UTF-8 string output by java will not carry BOM`

File: src/main/java/io/github/fvarrui/javapackager/utils/CommandUtils.java [hunk 1]
```diff
@@ -47,7 +45,7 @@ public static ExecutionResult executeWithResult(File workingDirectory, String ex
 
 		Process process = command.execute();
 
-		BufferedReader output = new BufferedReader(new InputStreamReader(process.getInputStream()));
+		BufferedReader output = new BufferedReader(new InputStreamReader(process.getInputStream(), Charset.forName("GBK")));
```
[the remark this answers is not in the document]
Comment (fvarrui) [reply src/main/java/io/github/fvarrui/javapackager/utils/CommandUtils.java:48]: Are you sure this won't present other problems with other language encodings?

[the remark this answers is not in the document]
Comment (lhDream) [reply src/main/java/io/github/fvarrui/javapackager/utils/CommandUtils.java]: Hi @fvarrui  
Do you have any questions?

**Comment by fvarrui** [comment]
Hi @lhDream!
Thanks for your PR ... I'll give it a try asap.

**Comment by fvarrui** [comment]
Please, take a look at this:

https://github.com/fvarrui/JavaPackager/blob/49528dc8a4a42e448f3a6797bdb70f52ad8b63a7/src/main/java/io/github/fvarrui/javapackager/utils/VelocityUtils.java#L65-L76

I reverted your changes in this part of the code, but keeping the possibility to render files with Velocity including the BOM ... We should only include the BOM in those files where it is essential, not in all files by default. Do you know which files really need it?

**Comment by lhDream** [comment]
Hi @fvarrui 
```
It is currently known that the iss.vtl file requires
```

**Comment by fvarrui** [comment]
Ok, done!

https://github.com/fvarrui/JavaPackager/blob/60bd8cb461eb212b309d9292fcd9963908a5ebf9/src/main/java/io/github/fvarrui/javapackager/packagers/GenerateSetup.java#L57-L59

https://github.com/fvarrui/JavaPackager/blob/60bd8cb461eb212b309d9292fcd9963908a5ebf9/src/main/java/io/github/fvarrui/javapackager/utils/VelocityUtils.java#L69-L77

Could you test branch `pr-272`?
Thanks!

**Comment by lhDream** [comment]
Hi @fvarrui 
Of course, but there is no such branch as pr-272

**Comment by fvarrui** [comment]
Yes, sorry, I mean 271 🙄

**Comment by lhDream** [comment]
@fvarrui 
I tested with pr-271 branch no problem.

**Comment by lhDream** [comment]
@fvarrui 
I tested with pr-271 branch no problem.

**Comment by fvarrui** [comment]
Merged into `devel`, ready to be released in the next version

**Comment by commi** [comment]
FYI: This change breaks with the non-unicode InnoSetup 5.x version... it cant read the iss file. 

This IS version is old, but this could be added to the documentation. (use 6.x oder 5.x Unicode)

**Pull request merged**